src/mozilla_sec_eia/models/sec10k/notebooks/exhibit21_layout_classifier.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.    for classifier, model in classifiers.items():
</pre><p>Do you think we should run cross validation of the models every time? Or should we just choose one (SVM performed the best). I guess it's fine that we just log all of them in MLFlow and then just choose one but could potentially use the wrong one in production.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/mozilla-sec-eia/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>